src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommand.cs

```diff
@@ -191,6 +192,10 @@ public MigrateRepoCommand() : base(
         name: "--no-ssl-verify",
         description: "Disables SSL verification when communicating with your Bitbucket Server/Data Center instance. All other migration steps will continue to verify SSL. " +
                      "If your Bitbucket instance has a self-signed SSL certificate then setting this flag will allow the migration archive to be exported.");
+    public Option<bool> UseGithubStorage { get; } = new(
```

Comment:
`UseGithubStorage` property also needs to be added to [MigrateRepoCommandArgs](https://github.com/github/gh-gei/blob/169c0c8faa1a7e05821b0da019455766c1aff8e1/src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandArgs.cs) plus some validations. For example it cannot be provide with both Azure storage connection string or AWS s3 stuff.